docs/en/setup/examples/grpc-load-balance-client/README.md

```diff
@@ -0,0 +1,84 @@
+# GRPC load balance client
+
+GRPC client support connect to multiple server address, and using round-robin to load-balance server before send each request.
+
+## Server Discovery
+
+Support two ways to locate the server list:
+1. Static server list: Define the server address list.
```

Comment:
Why do we need to mention `static list again`? I remember we used to have one already.

Reply:
Here is an example to let users know how to define multiple servers, `static server lists` and `Kubernetes selector` are two different ways.

Reply:
I see. Then you missed the menu update.

Reply:
Added.